.github/CODEOWNERS

```diff
@@ -167,3 +167,5 @@
 /src/automation/ @arrownj
 
 /src/rdbms-connect/ @mjain2 @Juliehzl
+
+/src/cosmosdb-continuous-backup/ @kavskalyan
```

Comment:
>@kavskalyan [](start = 33, length = 11)

Please add my and wmengmsft alias as well.

Reply:
Added you and wenbin